backend/db/migrations/20210101173805_add_duration_column_to_benchmarks.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE benchmarks ADD duration interval NOT NULL;
```

Comment:
This column was removed in #25.

Reply:
I'd prefer if we dropped `duration`. I thought it would be useful but later realised that mirage/index benchmarks already compute the duration (as the "time" value under "metrics).

Reply:
I'd prefer if we dropped `duration`. I thought it would be useful but later realised benchmarks like mirage/index compute their own duration (included in the "metrics" as "time").

Reply:
The duration might still be useful to figure out how stable the benchmarks are on autumn, we might not necessarily display it, but it can be used to figure out if the machine has to be optimised further.

Reply:
Usually the benchmarks will have a loop that they measure the duration of - the duration of the full benchmark (ie including init and shutdown) may not be designed to be stable, even if the main benchmark result is. So I’m not sure this value will be that useful to track if the benchmark itself doesn’t report it

Reply:
@tatchi There's no need to delete this column because it doesn't exist in production. The CI to apply migration's hasn't been set up yet, so I've been making the changes manually.

At this stage, given our priorities, I think the benchmark authors should measure the execution duration of the full benchmark themselves if they find it useful.

We can always add this later, I'm just not happy with having the `duration` column now because:

- It's not immediately useful.
- We might want to add other generic metrics such as container CPU and mem usage. This should be probably stored inside the `metrics` column with a special prefix such as `ocaml-benchmarks.cpu`, or something along these lines.